tests/spec/core/utils-spec.js

```diff
@@ -453,6 +454,42 @@ describe("Core - Utils", () => {
     });
   });
 
+  describe("htmlJoinAnd", () => {
+    it("joins with proper commas and 'and'", () => {
+      const div = document.createElement("div");
+      const render = hyperHTML.bind(div);
+
+      render`${utils.htmlJoinAnd([], item => hyperHTML`<a>${item}</a>`)}`;
+      expect(div.textContent).toBe("");
```

Comment:
We additionally need to check we get `<a>` as expected. Just `expect(div.getElementsByTagName("a").length).toBe(...)` would be fine.